examples/MultipleRedeemers.purs

```diff
@@ -0,0 +1,254 @@
+module Examples.MultipleRedeemers (threeRedeemerContract) where
+
+import Prelude
+
+import Cardano.Types.Value
```

Comment:
Why use `Cardano.Types.Value`? It should be possible to do everything with the public `Contract` interface